src/modules/Crop/Module.js

```diff
@@ -15,12 +15,7 @@
  */
 module.exports = function CropModule(options, UI) {
 
-  // TODO: we could also set this to {} if nil in AddModule.js to avoid this line:
-  options = options || {};
-
-  // Tell the UI that a step has been added
-  UI.onSetup(options.step); // we should get UI to return the image thumbnail so we can attach our own UI extensions
-
+  // we should get UI to return the image thumbnail so we can attach our own UI extensions
   // add our custom in-module html ui:
   if (options.step.inBrowser) var ui = require('./Ui.js')(options.step, UI);
```

Comment:
So if we wanted to customize the UI, this is a good example module. Maybe we should link into this from the README as an example of a customized UI?